config/google_managed_prometheus/README.md

```diff
@@ -0,0 +1,33 @@
+# Google Managed Prometheus
+
+The Prometheus Exporter can be used to send metrics to [Google Managed Prometheus](https://cloud.google.com/stackdriver/docs/managed-prometheus) (GMP). This directory contains sub directories with usercase specific configurations, all of which are compatible with GMP.
+
+## Google Cloud APIs
+
+Enable the following APIs.
+- Cloud Metrics
+
+To learn more about enabling APIs, see the [documentation](https://cloud.google.com/endpoints/docs/openapi/enable-api).
+
+## Prometheus
+
+It is assumed that you have Google's [Prometheus fork](https://github.com/GoogleCloudPlatform/prometheus/tree/main) running within your
```

Comment:
Does it not work with base prometheus? Just curious didn't know this.

Reply:
TLDR: Google's fork will scrape the Prometheus exporter (like regular Prometheus would) and it will forward those metrics to GMP.